bff/src/main/java/org/opennms/devjam2022/bff/BFFApplication.java

```diff
@@ -0,0 +1,112 @@
+/*******************************************************************************
+ * This file is part of OpenNMS(R).
+ *
+ * Copyright (C) 2022 The OpenNMS Group, Inc.
+ * OpenNMS(R) is Copyright (C) 1999-2022 The OpenNMS Group, Inc.
+ *
+ * OpenNMS(R) is a registered trademark of The OpenNMS Group, Inc.
+ *
+ * OpenNMS(R) is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as published
+ * by the Free Software Foundation, either version 3 of the License,
+ * or (at your option) any later version.
+ *
+ * OpenNMS(R) is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with OpenNMS(R).  If not, see:
+ *      http://www.gnu.org/licenses/
+ *
+ * For more information contact:
+ *     OpenNMS(R) Licensing <license@opennms.org>
+ *     http://www.opennms.org/
+ *     http://www.opennms.com/
+ *******************************************************************************/
+
+package org.opennms.devjam2022.bff;
+
+import static akka.http.javadsl.server.PathMatchers.longSegment;
+
+import java.io.IOException;
+import java.util.Optional;
+import java.util.concurrent.CompletableFuture;
+import java.util.concurrent.CompletionStage;
+
+import akka.actor.typed.ActorSystem;
+import akka.actor.typed.javadsl.Behaviors;
+import akka.http.javadsl.Http;
+import akka.http.javadsl.ServerBinding;
+import akka.http.javadsl.model.HttpEntity;
+import akka.http.javadsl.model.StatusCodes;
+import akka.http.javadsl.server.AllDirectives;
+import akka.http.javadsl.server.Route;
+
+public class BFFApplication extends AllDirectives {
+  public static void main(String[] args) throws IOException {
+    ActorSystem<Void> actorSystem = ActorSystem.create(Behaviors.empty(), "bff-actor-system");
+
+    final Http http = Http.get(actorSystem);
+
+    BFFApplication app = new BFFApplication();
+
+    final CompletionStage<ServerBinding> binding = http.newServerAt("localhost", 8081)
+        .bind(app.createRoute());
+
+    System.out.println("Server online at http://localhost:8081/\nPress RETURN to stop...");
+    System.in.read();
```

Comment:
Should be OK for now, but I would try to eliminate this one ASAP and implement something more proper